### ISSUE TITLE
Search in Blog (feature)

### ISSUE THREAD

**Issue by rahuldotbiz** [W3Layouts/delog#22]
If this can have the amazing in blog search icon so we can search through posts.

**Comment by morbishreejioff** [comment]
I have the same request, If possible, have a search functionality in this great blog.
second Pagination and instead of adding date in card, can we add tags or category? Its much useful if its possible,
Thanks

**Comment by rahuldotbiz** [comment]
I guess there are many starts having more than these functionalities.

**Comment by morbishreejioff** [comment]
just like many starts? can you give me any references or links?

**Comment by rahuldotbiz** [comment]
Sure, will do that.

You need for Gatsby only? Or fot any SSGs?

**Comment by morbishreejioff** [comment]
yes, my first priority is Gatsby, otherwise if functions and facilities are great then I can switch to your guidance too
Thanks for the help

**Comment by morbishreejioff** [comment]
Hi, Please kindly reply if possible,
Thanks 🙏🙏🙏

**Comment by rahuldotbiz** [comment]
Give me 10mins please.

**Comment by rahuldotbiz** [comment]
##### Gatsby

- [Gatsby Starter Novela](https://github.com/narative/gatsby-starter-novela)
- [Gatsby Starter Nehalem](https://github.com/nehalist/gatsby-starter-nehalem)
- [gatsby-starter-developer-diary](https://github.com/willjw3/gatsby-starter-developer-diary)
- [gatsby-starter-developer-blog](https://github.com/lgcolella/gatsby-starter-developer-blog)

--- 

##### Eleventy 

(I prefer 11ty over gatsby)
- [11r](https://github.com/reeseschultz/11r)
- BEST [Hylia](https://github.com/hankchizljaw/hylia)
- [EleventyOne](https://github.com/philhawksworth/eleventyone)

**Comment by morbishreejioff** [comment]
Hello,
Thanks for the reply,
I'm looking theme have
-Search function (Ajax Search/doc search, not algolia its too costly)
-Tag and can be filtered by tags
-Pagination
-card view like elements.envato.com cards for post just like your theme
I think its simple and easy requirements still not fully satisfied with this.

Ghost is alternate but, it's not capable with Gatsby-react.
I hope now you get this my view,
Thanks for the help🙏🙏🙏

**Comment by rahuldotbiz** [comment]
No problem, I understand. But adding these features are just simple so
simple. There are lots of tutorials on YouTube.

**Comment by morbishreejioff** [comment]
Hi,
That is right, tutorial is available but, I'm not a coder.
Basically, my profile is Graphics Designer.
I can design beautifully but, I cannot code fully 
I am aware of react like each component should include import-export
I can read code and can understand but, fully execute all things,
that's why I find available theme,
Thanks for help 🙏🙏🙏

**Comment by rahuldotbiz** [comment]
No problem, I understand. If you want I can help you integrate features...

**Comment by morbishreejioff** [comment]
Hi, Thanks for the help offering
Can you please add these features to your theme?
so, a person likewise I can use, what I am telling is there no fully functional theme is available so,
people can get benefit from yours if you available such kind of theme?

I hope you get my points,
Let me know your view,
Thanks

**Comment by rahuldotbiz** [comment]
I can try 😀

**Comment by morbishreejioff** [comment]
Many Thanks 🙏🙏🙏

**Comment by talmax1124** [comment]
@rahxuls Did you finish it? I have almost the same request but I am using Algolia. 

I need to do the part where he changes the things to be filtered but don't understand what I need to change in this actual template would someone help me?

Check out this tutorial: https://youtu.be/VSkXyuXzwlc?t=2375
